cdap-client-tests/src/test/java/co/cask/cdap/client/WorkflowClientTestRun.java

```diff
@@ -89,15 +89,10 @@ public void testWorkflowClient() throws Exception {
       NamespaceId.DEFAULT.app(AppWithWorkflow.NAME).workflow(AppWithWorkflow.SampleWorkflow.NAME);
 
     programClient.start(workflowId, false, runtimeArgs);
+    assertProgramRunning(programClient, workflowId);
+    assertProgramRuns(programClient, workflowId, ProgramRunStatus.RUNNING, 1);
     programClient.waitForStatus(workflowId, ProgramStatus.STOPPED, 60, TimeUnit.SECONDS);
```

Comment:
It is technically needed since wait for status takes in a configurable amount of time (60 seconds in this case). I thought it was best to use that function for those longer waits, and wait for runs with a shorter, non-configurable time for simplicity